Library/Homebrew/utils/pypi.rb

```diff
@@ -208,30 +208,21 @@ def update_python_resources!(formula, version: nil, package_name: nil, extra_pac
     @pipgrip_installed ||= Formula["pipgrip"].any_version_installed?
     odie '"pipgrip" must be installed (`brew install pipgrip`)' unless @pipgrip_installed
 
-    found_packages = []
-    input_packages.each do |package|
-      ohai "Retrieving PyPI dependencies for \"#{package}\"..." if !print_only && !silent
-      pipgrip_output = Utils.popen_read Formula["pipgrip"].bin/"pipgrip", "--json", "--no-cache-dir", package.to_s
-      unless $CHILD_STATUS.success?
-        odie <<~EOS
-          Unable to determine dependencies for \"#{package}\" because of a failure when running
-          `pipgrip --json --no-cache-dir #{package}`.
-          Please update the resources for \"#{formula.name}\" manually.
-        EOS
-      end
-
-      JSON.parse(pipgrip_output).to_h.each do |new_name, new_version|
-        new_package = Package.new("#{new_name}==#{new_version}")
+    input_packages = input_packages.map(&:to_s)
```

Comment:
```suggestion
```

I don't think this is needed. The only other uses of `input_packages` are either already calling `input_packages.map(&:to_s)` or call `input_packages.join(" ")` which intenrally calls `to_s`.

So I'd say either remove this line or just use `* input_packages` on the next line.

Reply:
Nice catch, thanks Rylan!
Fixed.